tests/docker/tests_integration/run_tests.sh

```diff
@@ -16,4 +23,10 @@ pwd
 dart bin/main.dart -m production -r maintenance --apply-migrations
 
 # Run tests
+echo "### Running tests"
 dart test test_integration --concurrency=1
+checkLastExitCode
+
+cd ../serverpod_test_module/serverpod_test_module_server/
+echo $(pwd)
+dart test ./test/integration
```

Comment:
Question: Don't we need to apply the migrations specifically for the test module?

Reply:
We don't have any database models in this module as of yet. If we start adding that then it would be needed.

Reply:
Hmm, I think we should be OK either way since we don't give any warnings if "additional" tables are found in the database and all the modules tables are applied together with the server.

Reply:
Ah, right!